### PR TITLE
[Assignment 1] Create a basic HTTP server #1

### PR DESCRIPTION
**Pull request by zoe-1**
A hapijs server with the below characteristics:
- lib folder
- package.json file
  Package file defines hapijs as a project dependency.
- An initial index.js file
- Server is created on port 8000.
- A route accepting /version requests and replies
  { "version":"0.0.1" } json payload.
  Above version response comes from package.json file.

### CODE REVIEW

**Comment by zoe-1** [comment]
This projects tab space should be four spaces.  
Githubs default is eight spaces for a tab.
Please append "?ts=4" to the end of the url then spacing will align the style guide requirements.
I hope this is acceptable.

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// Basic hapijs http server.
+
+var Hapi = require('hapi');
+
+// Utility methods for the hapi ecosystem.
+var Hoek = require('hoek');
+
+// Load needed internal settings
+var internals = {
+	pkg:require('../package.json')
```
Comment (AdriVanHoudt) [inline lib/index.js]: space after :

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+// Configure server to use specific port.
+server.connection({ port: 8000 });
+
+
+// Create a route responding to requests.  
+// Visit localhost:8000/version and see response.
+server.route({
+
```
Comment (AdriVanHoudt) [inline lib/index.js]: no new line required here

File: lib/index.js (hunk 1 above)
Comment (zoe-1) [inline lib/index.js]: @AdriVanHoudt Thank you for the reviewing the code and making recommendations. 
I made the changes.

File: lib/index.js (hunk 2 above)
Comment (zoe-1) [inline lib/index.js]: Thanks. made the changes.

File: lib/index.js (hunk 1 above)
Comment (AdriVanHoudt) [inline lib/index.js]: @zoe-1 np you can read the full style guide [here](http://hapijs.com/styleguide)

**Comment by AdriVanHoudt** [comment]
@zoe-1 I'm afraid that the indentation is required to be 4 spaces, no one wants to continuously add ?ts=4 to every pr and such

**Comment by zoe-1** [comment]
@AdriVanHoudt OK. I made the tabs to 4 spaces.

**Comment by AdriVanHoudt** [comment]
@zoe-1 :+1:

**Comment by zoe-1** [comment]
@AdriVanHoudt Thanks again for the review. :-)

**Comment by AdriVanHoudt** [comment]
@zoe-1 np :D

**Pull request closed without merging**